app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php

```diff
@@ -151,10 +151,7 @@ public function execute()
                 $status = (int) $this->_subscriberFactory->create()->subscribe($email);
                 $this->messageManager->addSuccessMessage($this->getSuccessMessage($status));
             } catch (LocalizedException $e) {
-                $this->messageManager->addExceptionMessage(
-                    $e,
-                    __('There was a problem with the subscription: %1', $e->getMessage())
-                );
+                $this->messageManager->addErrorMessage(__('This email address is already subscribed.'));
```

Comment:
I think the throw exception on line 146 should be replaced with this messageManager call. Here several different exceptions are possible (i.e. validateEmailFormat also throws LocalizedException)

Reply:
@sivaschenko yes you are right, I will change this too.